metron-platform/metron-enrichment/src/main/java/org/apache/metron/enrichment/bolt/GenericEnrichmentBolt.java

```diff
@@ -161,6 +167,7 @@ protected void initializeStellar() {
     stellarContext = new Context.Builder()
                          .with(Context.Capabilities.ZOOKEEPER_CLIENT, () -> client)
                          .with(Context.Capabilities.GLOBAL_CONFIG, () -> getConfigurations().getGlobalConfig())
+                         .with(Context.Capabilities.GEO_IP, () -> GeoLiteDatabase.INSTANCE)
```

Comment:
@cestella Capability was left over from earlier testing.  Everything referred to the static instance anyway for precisely that reason.  Dropped it entirely and removed from the enum.

@dlyle65535 There's not a geo specific bolt, it's an instance of GenericEnrichment. To the best of my knowledge (and correct me if I'm wrong), you can't grab global configs in the adapters.  This specific one gets dropped, but the prepare() method references the static INSTANCE Casey refers to.  Is there somewhere that could live?